.cirrus.yml

```diff
@@ -0,0 +1,19 @@
+freebsd_task:
+  freebsd_instance:
+    matrix:
+    - image: freebsd-13-2-release-amd64
+  sysctl_script:
```

Comment:
Would you mind to add a comment to explain these commands? Maybe add add these two links:

```
# https://github.com/cirruslabs/cirrus-ci-docs/issues/483
# https://reviews.freebsd.org/D41751
```